src/generators/app-lit-element-ts/templates/_my-app.stories.js

```diff
@@ -0,0 +1,25 @@
+import { html } from 'lit-html';
+import '../out-tsc/src/<%= tagName %>.js';
+
+export default {
+  title: '<%= className %>',
+  component: '<%= tagName %>',
+  argTypes: {
+    backgroundColor: { control: 'color' },
+  },
+};
+
+function Template({ title, backgroundColor }) {
+  return html`
+    <<%= tagName %>
+      style="--<%= tagName %>-background-color: ${backgroundColor || 'white'}"
+      .title=${title}
+    >
+    </<%= tagName %>>
+  `;
+}
+
+export const App = Template.bind({});
```

Comment:
In the docs you use this syntax:
```
const Button = ({ backgroundColor = 'white', text }) => {
  return `
    <button type="button" style="background-color: ${backgroundColor}">
      ${text}
    </button>
  `;
};

export const ButtonA = args => Button(args);
ButtonA.args = {
  text: 'Button A',
};
```
Is there a preference for `.bind()` that we should share there as well?

Reply:
Good one, I think both work. The `.bind` is a bit odd, but that's how they document it on the storybook side. Perhaps best for us to follow that as well?

Reply:
I feel like the `.bind()` variant caused issues when I wanted to use the same stories in my tests, but I may be making that up. If they're using that syntax everywhere it's gonna save us questions to use it everywhere, too. Regardless of how weird it looks.

Reply:
I will update the docs as well